test/13_arraycommon.jl

```diff
@@ -1638,8 +1638,8 @@ end
         @test sprint((io,a)->show(io, "text/plain", a), x) ==
             """
             3-element $CategoricalArray{$(Union{Missing,Int}),1,UInt32}:
-             2      
-             1      
+             2
+             1
```

Comment:
This is the reason for the 1.0 failure. :-D
```suggestion
             2      
             1      
```